Source/Core/Core/CoreParameter.cpp

```diff
@@ -420,10 +420,13 @@ IniFile SCoreStartupParameter::LoadGameIni(const std::string& id, int revision)
 void SCoreStartupParameter::LoadGameIni(IniFile* game_ini, const std::string& path,
                                         const std::string& id, int revision)
 {
-	// INIs that match all regions
+	// INIs that match all regions (Without publisher codes)
 	if (id.size() >= 4)
 		game_ini->Load(path + id.substr(0, 3) + ".ini", true);
 
+	// INIs that match all regions (With publisher codes)
+	game_ini->Load(path + id.substr(0, 3) + "-" + id.substr(4) + ".ini", true);
```

Comment:
Handling malformed IDs better is my reason for wanting to use `>= 5`. I don't see any advantages with using `== 6`.